packages/authenticator/amplify_authenticator/lib/src/widgets/form.dart

```diff
@@ -663,3 +688,11 @@ class ConfirmVerifyUserForm extends AuthenticatorForm {
   AuthenticatorFormState<ConfirmVerifyUserForm> createState() =>
       AuthenticatorFormState<ConfirmVerifyUserForm>();
 }
+
+String _duplicateFieldLog(String alias, String field) {
+  return 'SignUpForm contains `SignUpFormField.$field`, but Amplify Auth is '
+      'configured to use $alias as a username. This will result in duplicate '
+      '$alias fields. Consider removing `SignUpFormField.$field`, or changing '
+      'how users sign in. For more info, see: '
+      'https://pub.dev/documentation/amplify_authenticator/latest/amplify_authenticator/SignUpFormField/username.html';
```

Comment:
Not sure how helpful this link is

Reply:
Would this be more relevant? https://ui.docs.amplify.aws/flutter/connected-components/authenticator/customization#sign-up-fields

Reply:
This doc comment is what I was trying to point the user to. In particular, the last two paragraphs.

> Creates a username component based on your app's configuration.
>
> The look and behavior of this widget will depend on your app's auth configuration. The usernameAttributes will be used to determine if the field is a standard username field, and email field, or a phone number field.
> 
> If your app is configured to use email or phone, the value provided for this widget will be saved as the user's email or phone number in addition to their username.